content/app/github/window-helper.js

```diff
@@ -33,6 +33,15 @@ define("content/app/github/window-helper", function(require, exports, module) {
 				return new Box(0, 0, width, height);
 			},
 
+			/**
+			 * Scrolls vertically document body.
+			 * @param  {Number}		scrollDelta
+			 */
+			scrollTop : function(scrollDelta) {
```

Comment:
Scrolling by delta requires that caller know about current scroll position. I think that it is better that we make this function to scroll to absolute position. As I see in code that is using this method it would be simpler is you call scrolling by absolute position.
Instead of:
var viewportBox = WindowHelper.getViewportBoundingBox();
var fileBox = desiredFileWrapper.boundingBox;
var scrollDelta = fileBox.top - viewportBox.top;
scrollDelta -= GotoFileCommand.MARGIN_TOP;
WindowHelper.scrollTop(scrollDelta);
It would be:
var fileBox = desiredFileWrapper.boundingBox;
WindowHelper.scrollTop(fileBox.top - GotoFileCommand.MARGIN_TOP);